FEXCore/Source/Interface/Core/JIT/JIT.cpp

```diff
@@ -377,17 +368,14 @@ void Arm64JITCore::Op_Unhandled(const IR::IROp_Header* IROp, IR::NodeID Node) {
       const auto Src2 = GetVReg(IROp->Args[1].ID());
 
       ldrh(ARMEmitter::WReg::w0, STATE, offsetof(FEXCore::Core::CPUState, FCW));
-      umov<ARMEmitter::SubRegSize::i64Bit>(ARMEmitter::Reg::r1, Src1, 0);
-      umov<ARMEmitter::SubRegSize::i16Bit>(ARMEmitter::Reg::r2, Src1, 4);
+      mov(ARMEmitter::VReg::v0.Q(), Src1.Q());
+      mov(ARMEmitter::VReg::v1.Q(), Src2.Q());
```

Comment:
This isn't valid on arm64ec, Src2 could be allocated to v0 - you'll need to copy to vtmp before hand as is done for some others with tmp